docs/src/examples/components/Accordion/Usage/AccordionPanelCustomTitleExample.shorthand.tsx

```diff
@@ -19,6 +19,29 @@ class AccordionPanelCustomTitleExample extends React.Component {
           content: 'Here is a list of warnings discovered.',
         },
       },
+      {
+        title: {
+          content: (
+            <div>
+              <strong>Someone</strong> has added <strong>Someone Else</strong> to the team
+            </div>
+          ),
+          icon: { name: 'participant-add', variables: { outline: true } },
+        },
+        content: {
+          key: 'peopleadded',
+          content: (
+            <div>
+              <div style={{ padding: '0 0 .5em 0' }}>
```

Comment:
please do not change example styles to match specific theme;

I'd suggest reverting these changes in this file and create a prototype (use #931 as a reference on how to create one)

Reply:
What about creating a new example under Usages? If it's an example, then screenshot tests will capture any regressions - I don't think prototypes have that same benefit

Reply:
> What about creating a new example under Usages?

I also don't think it's a good practice.

The point of the Examples section is to provide **simple and generic code snippets** to show clients how to **use** our components. It is not intended to show them a very particular set of customizations to achieve a very particular flavor for that component.

> If it's an example, then screenshot tests will capture any regressions - I don't think prototypes have that same benefit

They don't but these screenshots tests are not meant to capture regressions for styles inserted in the examples themselves, but for the whole theme (so for styles that are introduced in the top level theme object for the theme that is currently displayed in the docs site). Hope this makes sense.